src/rest_framework_jwt/blacklist/views.py

```diff
@@ -0,0 +1,26 @@
+from django.db import IntegrityError
+
+from rest_framework.exceptions import ValidationError
+from rest_framework.viewsets import ModelViewSet
+
+from rest_framework_jwt.blacklist.models import BlacklistedToken
+
+from ..authentication import JSONWebTokenAuthentication
+from .permissions import CanBlacklist
+from .serializers import BlacklistTokenSerializer
+
+
+class BlacklistView(ModelViewSet):
+    permission_classes = (CanBlacklist, )
+    authentication_classes = (JSONWebTokenAuthentication, )
```

Comment:
Is this authentication necessary? What happens if someone overrides it and sets it in the `JWT_AUTH` settings?